payment_payzen/data/mail_templates_data.xml

```diff
@@ -0,0 +1,96 @@
+                </t>
+                auprès de<t t-out="object.company_id.name"/>.
+            </p>
+
+            <p t-if="object.recurring_payment_months > 1">
+                <t t-if="object.recurring_payment_months">
+                    Payable en <t t-out="object.recurring_payment_months"/> fois
+                    <br/>
+                </t>
+                <t t-if="object.recurring_first_payment_amount">
+                    Premier paiement de
+                    <t t-out="format_amount(object.recurring_first_payment_amount, object.currency_id)"/>
+                    le
+                    <t t-out="format_date(object.date_order)"/>
+                </t>
+                <t t-else="">
+                    Avec des paiements à hauteur de
+                    <t t-out="format_amount((object.recurring_payment_monthly_amount), object.currency_id)"/>
+                </t>
+                <br/>
+                <t t-if="object.recurring_second_payment_amount">
+                    La première mensualité sera de
+                    <t t-out="format_amount(object.recurring_second_payment_amount, object.currency_id)"/>
+                    pour faire la balance,
+                    <t t-if="object.recurring_second_payment_date">
+                        le
+                        <t t-out="format_date(object.recurring_second_payment_date)"/>
+                        <br/>
+                    </t>
+                    Ensuite, paiments de
+                    <t t-out="format_amount((object.recurring_payment_monthly_amount), object.currency_id)"/>
+                </t>
+                <t t-else="">
+                    Prochain paiement de
+                    <t t-out="format_amount((object.recurring_payment_monthly_amount), object.currency_id)"/>
+                    <t t-if="object.recurring_second_payment_date">
+                        le
+                        <t t-out="format_date(object.recurring_second_payment_date)"/>
+                        <br/>
+                    </t>
+                </t>
+            </p>
+
+            <br/>
+
+            <p t-if="is_online and not ctx.get('proforma')">
+                <center>
+                    <a style="background-color: #1abc9c; padding: 20px; text-decoration: none; color: #fff; border-radius: 5px; font-size: 16px;"
+                       class="o_default_snippet_text" t-attf-href="{{ access_url }}">Je règle en ligne
+                    </a>
+                    <br/>
+                    <br/>
+                    <br/>
+                    <span style="color:#888888">(Devis en pièce-jointe)</span>
+                </center>
+            </p>
+
+            <br/>
+
+            <p style="color:#888888;">
+                <t t-if="object.user_id and object.user_id.signature">
+                    <t t-out="object.user_id.signature"/>
+                </t>
+            </p>
+        </field>
+    </record>
+</odoo>
```

Comment:
No new line at End of File
There are a few more in other files, I let you check to avoid comenting everywhere
(pre-commit really saves us for this kind of things haha)

Reply:
No pre-commit on this repo, and I won't add it to stay closer to the [original source code](https://github.com/lyra/plugin-odoo/releases/tag/v4.0.0) (will make it much easier to merge and upgrade in the future).
Therefore I'm not even sure if it makes sense to fix EOF lines.